### PR TITLE
Follow/unfollow functionality to button to publictimeline, missing fr…

### PR DESCRIPTION
**Pull request by Maquidos**
…om privatetimeline.

Co-authored-by: chat.openai.com

### CODE REVIEW

File: src/Chirp.Infrastructure/AuthorRepository.cs [hunk 1]
```diff
@@ -58,32 +58,81 @@ public async Task CreateAuthor(string name, string email)
...
+    {
+        throw new Exception("User already follows this author");
+    }
+}
 
-    public async Task UnFollowAnAuthor(string followingEmail, string unFollowingEmail)
+public async Task UnFollowAnAuthor(string followingEmail, string unFollowingName)
+{
+    var followingAuthor = await _context.Authors.FirstOrDefaultAsync(a => a.Email == followingEmail);
+    var unFollowingAuthor = await _context.Authors.FirstOrDefaultAsync(a => a.Name == unFollowingName);
```
Comment (allanzigge) [inline src/Chirp.Infrastructure/AuthorRepository.cs:90]: Hvorfor bruges email(l. 89) og derefter name

File: src/Chirp.Web/Pages/Public.cshtml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ﻿@page "/"
 @model Chirp.Web.Pages.PublicModel
+@inject IAuthorRepository AuthorRepository
```
Comment (palleBrandt) [inline src/Chirp.Web/Pages/Public.cshtml:3]: I think this should be removed

**Review by palleBrandt — CHANGES_REQUESTED** [review]
I made a comment with the requested change

**Review by palleBrandt — APPROVED** [review]
Looks good

**Pull request merged**